cub/cub/device/dispatch/dispatch_scan.cuh

```diff
@@ -286,15 +342,16 @@ struct DispatchScan
       int scan_sm_occupancy;
       error = CubDebug(MaxSmOccupancy(scan_sm_occupancy, // out
                                       scan_kernel,
-                                      Policy::BLOCK_THREADS));
+                                      policy.Scan().BlockThreads()));
       if (cudaSuccess != error)
       {
         break;
       }
 
+      // TODO(ashwin): should this come from the launcher factory instead?
```

Comment:
This comment seems stale, please remove it.
```suggestion
```

Reply:
Done.